graphs/eacs/src/test/kotlin/com/anaplan/engineering/azuki/graphs/eacs/BEH6.kt

```diff
@@ -0,0 +1,142 @@
+package com.anaplan.engineering.azuki.graphs.eacs
+
+import com.anaplan.engineering.azuki.core.runner.Eac
+import com.anaplan.engineering.azuki.core.system.BEH
+import com.anaplan.engineering.azuki.graphs.adapter.api.GraphBehaviours
+import com.anaplan.engineering.azuki.graphs.dsl.GraphScenario
+import com.anaplan.engineering.azuki.graphs.graphA
+
+@BEH(GraphBehaviours.HasCycle, GraphBehaviours.CreateDirectedGraph, """
+    Determine if one or more cycles exist in graph
```

Comment:
⛏️ It should be clear from the requirement what is the expected input and what is not.
```suggestion
    Determine if one or more cycles exist in a given graph
```

Reply:
⛏️ The phrase is also reads a little odd, so I would suggest something like

> Determine if the given directed graph contains one or more cycles

unless there is a good reason why the cycles should be the subject.